what-the-stack/build.gradle

```
@@ -36,7 +36,11 @@ android {
     }
 
     buildFeatures {
-        viewBinding true
+        compose true
+    }
+
+    composeOptions {
+        kotlinCompilerExtensionVersion "1.0.5"
```

After making the change in libs.version.toml

```suggestion
        kotlinCompilerExtensionVersion libs.versions.compose.get()
```

---

TIL. Didn't know the DSL for versions catalog supported such operations. Fixed now.